### DISCUSSION TITLE
Radio test

### DISCUSSION THREAD

**Discussion by jwhellenthal** [micropython/micropython#10797] [STM32 / Pyboard] [answered]
We have  a SF6W Pyboard module to be used within a product. The test house wants us to tune/tweak several settings on the Infineon 1DX module for radio compliancy testing. They ask us to do the following:
        WIFI 2.4 GHz  Radio test mode
        ·        Frequency selection, no modulation
        ·         Output power selection
        ·        ….
No complete RED test is needed just some means to check if the module in the product housing is functioning  as expected.  
So basically we need some means to do radio tests on the Pyboard 1DX module.  The challenge is that there is no direct connection of the 1DX module with the outside world. There are some tools for testing but it is not clear how to use them in combination with the Pyboard.  So basically; 
Is there anyone who has experience addressing the 1DX module in combination with the Pyboard for radio tests?

Best regards

**Comment by dpgeorge** [comment, marked as answer]
As part of the FCC testing of the PYBD-SFxW series, we needed to provide the testing lab with special firmware that allowed the FCC tests to be carried out.  That was separate manufacturing firmware for WiFi and Bluetooth.

We can provide you with this firmware.  Please contact me at dpgeorge -at- georgerobotics.com.au
